graylog2-web-interface/src/views/logic/views/UpdateSearchForWidgets.js

```diff
@@ -0,0 +1,29 @@
+// @flow strict
+import { get } from 'lodash';
+import View from 'views/logic/views/View';
+import SearchTypesGenerator from 'views/logic/searchtypes/SearchTypesGenerator';
+
+
+const UpdateSearchForWidgets = (view: View): View => {
+  const states = view.state;
```

Comment:
Should this be destructed? 
```suggestion
  const { state: states } = view;
```

And if we have to change the name of the constant, should we just update the name of the key in the object to be plural as well?

Reply:
The reason why it is called states here, is to clarify that it is something to iterate over. But to have it as state is also valid. So no I would not rename it in the object.

Reply:
I agree with plural being a an easy identifier that it is iterable, but I just don't understand why we are renaming the variable in the middle.
If `view.state` is going to be defined as an object or array, then I don't see why we just don't call it `view.states` from the get-go instead of renaming it.